src/routes/web.php

```diff
@@ -1,20 +1,23 @@
 <?php
 
-$url = config('scheduler.url');
-$controller = 'MatviiB\\Scheduler\\Controllers\\SchedulerController';
+Route::group(['middleware' => config('scheduler.middleware')], function () {
+    $url = config('scheduler.url');
+    $controller = 'MatviiB\\Scheduler\\Controllers\\SchedulerController';
+    //
+    Route::get($url, ['as' => "$url.index", 'uses' => "$controller@index"]);
 
-Route::get($url, "$controller@index")->name("$url.index");
+    Route::get("$url/edit/{task}", ['as' => "$url.edit", 'uses' => "$controller@edit"]);
 
-Route::get("$url/edit/{task}", "$controller@edit")->name("$url.edit");
+    Route::patch("$url/update/{task}", ['as' => "$url.update", 'uses' =>  "$controller@update"]);
 
-Route::patch("$url/update/{task}", "$controller@update")->name("$url.update");
+    Route::get("$url/toggle/{task}", ['as' => "$url.toggle", 'uses' => "$controller@toggle"]);
 
-Route::get("$url/toggle/{task}", "$controller@toggle")->name("$url.toggle");
+    Route::get("$url/run/{task}", ['as' => "$url.run", 'uses' => "$controller@run"]);
 
-Route::get("$url/run/{task}", "$controller@run")->name("$url.run");
+    Route::get("$url/create", ['as' => "$url.create", 'uses' => "$controller@create"]);
 
-Route::get("$url/create", "$controller@create")->name("$url.create");
+    Route::post("$url/store", ['as' => "$url.store", 'uses' => "$controller@store"]);
 
-Route::post("$url/store", "$controller@store")->name("$url.store");
+    Route::delete("$url/delete", ['as' => "$url.delete", 'uses' => "$controller@delete"]);
 
```

Comment:
please do something with case if $url contains '/' and   ` 'as' `will be as "module/scheduler.delete". And I preferred when name is set as `->name('foo.bar')`. Please. Thanks

Reply:
We can replace it with syntax like `strreplace('/', '.', $url)`.
is this okay?
Thanks.

Reply:
yes, that's fine

Reply:
Okay.
I did that.
Thanks.